### ISSUE TITLE
Help with installation

### ISSUE THREAD

**Issue by maikocosta** [Johni0702/mumble-web#51]
Hello
How do I install mumble-web? I've tried every way and I can not.
Got a step by step?

**Comment by nsmith1024** [comment]
i have the same problem i use npm to install it, and also websocketfy. Then i use a web browser to try to connect like this http://www.example.com:64737 but it never connects

**Comment by dirkk0** [comment]
No, you need to use ssl, but you can use self-signed certificates.

I just verified this on Mac OSX.
- download Murmur-OSX-Static-1.3.0.tar.bz2 from https://www.mumble.info/downloads/
- enter the folder and execute `./murmurd -fg -ini murmur.ini`

Now the server should be running and connectable with any mumble client.

Then, install mumble-web:

    git clone https://github.com/johni0702/mumble-web
    cd mumble-web
    npm install
    npm run build

(or with yarn) as stated in the [installation](https://github.com/Johni0702/mumble-web#installing)

Download or clone the [websockify](https://github.com/novnc/websockify) python project.
(websockify-js unfortunately does not work).

    git clone https://github.com/novnc/websockify.git
    cd websockify
    python setup.py install

If you don't have key and cert, create these with

    sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./mykey.key -out mycert.crt

Now start websockify:

     sudo websockify --cert=mycert.crt --key=mykey.key --ssl-only --ssl-target --web=../dist 443 localhost:64738

Open a browser with https://localhost:443 and connect.
